LICENSE.md

```diff
@@ -270,3 +270,16 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 ### contextMenu [2.1.1](https://github.com/swisnl/jQuery-contextMenu)
 MIT licensed
+
+--------------------------------------------------------------------------------
+
+### FileSaver [1.1.20160328](https://github.com/eligrey/FileSaver.js/)
+### Blob [2014-07-24](http://purl.eligrey.com/github/Blob.js/blob/master/Blob.js)
```

Comment:
Yeah, but for consistency, let's have the same license repeated with
each package heading a repetition of the license.

On (May-02-16|20:20), Joshua Shorenstein wrote:
> > @@ -270,3 +270,16 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
> >  
> >  ### contextMenu [2.1.1](https://github.com/swisnl/jQuery-contextMenu)
> >  MIT licensed
> > +
> > +--------------------------------------------------------------------------------
> > +
> > +### FileSaver [1.1.20160328](https://github.com/eligrey/FileSaver.js/)
> > +### Blob [2014-07-24](http://purl.eligrey.com/github/Blob.js/blob/master/Blob.js)
> 
> It's the same license since it's made by the same guy, which is why I have the two together. How would you like me to break it up?
> 
> ---
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/466/files/8c43ce45e9e5ab5b86574f7bb97dd4257117935e#r61834847